21340.0.txt

```diff
@@ -0,0 +1,66 @@
+@0
+MOV UP, DOWN
+#MOVE INPUT DOWN
+
+@1
+
+
+@2
+
+
+@3
+
+
+@4
+MOV UP, DOWN
+#MOVE DOWN
+
+@5
+MOV UP, RIGHT
+#MOVE DOWN
+
+@6
+S: MOV LEFT, ACC
+#STORE IN ACC
+ MOV ACC, RIGHT
+#MOVE ACC RIGHT
```

Comment:
Comments like this (which are just a restatement of the code) aren't particularly helpful. Always better to focus on "why" rather than "what"